src/app/components/Article.jsx

```diff
@@ -24,8 +54,8 @@ class Article extends Component {
 
     try {
       const response = await fetch(url);
-      const data = await response.json();
-      return { data };
+      const { model } = await response.json();
```

Comment:
I presume you've left this as scenario-01 to a) keep this PR small and b) not start with a more complex example. However we should probably have an issue that we need to iterate the logic above until it works on our production examples.

Furthermore we need an issue to remove this hardcoding.

Given that this PR starts that stream of work I suggest these and other related issues are written up now.